### DISCUSSION TITLE
Any success using sass variables and storybook together?

### DISCUSSION THREAD

**Discussion by Seanitzel** [quasarframework/quasar#11537] [General - Components / Directives / etc] [answered]
I want to use Quasar with storybook.

I found this template - https://github.com/badsaarow/quasar2-storybook-boilerplate
Storybooks runs there but the components are not using any sass variables.

As soon  as a sass variable is used in a component storybook fails to start and throws an error that the variable is not defined.
I tried importing Quasar sass files and creating my own files that declare the variables and importing them, but no luck.

I Created this repo which reproduces the problem - https://github.com/Seanitzel/Quasar-V2-Storybook

Does it work for anyone?

**Comment by jclaveau** [comment]
Yes sir! 

With Storybook 7, scss support and Chromatic https://github.com/quasarframework/quasar/issues/11654#issuecomment-1618295838

**Comment by Seanitzel** [comment]
That's awesome!
Even though i already migrated away from Quasar sass variables a long time ago because of this 😆

**Reply by jclaveau** [reply, marked as answer]
Thank you! I just opened a dedicated feature request in case the Quasar team don't find it out of scope: https://github.com/quasarframework/quasar/discussions/16037

Feel free to upvote!
